addons/website_hr_recruitment/views/website_hr_recruitment_templates.xml

```diff
@@ -769,21 +769,21 @@
                 <a t-attf-href="/jobs?all_departments=1#{non_department_params}"
                     t-attf-class="dropdown-item d-flex align-items-center justify-content-between nav-link#{'' if department_id or is_other_department else ' active'}">
                     All Departments
-                    <span t-attf-class="badge rounded-pill#{' bg-primary' if department_id or is_other_department else ' bg-light text-primary'} ms-2" t-out="count_per_department.get('all', '0')"/>
+                    <span t-attf-class="badge rounded-pill text-bg-primary ms-2" t-out="count_per_department.get('all', '0')"/>
```

Comment:
Same question as in other places.. no need for differentiation?

Reply:
Answered [here](https://github.com/odoo/odoo/pull/162935#discussion_r1706970291)